src/NewCommand.php

```diff
@@ -13,7 +13,7 @@ class NewCommand extends Command
 {
     const LARAVEL_CURRENT_LTS_VERSION = '6.*';
     const LARAVEL_CURRENT_LATEST_VERSION = '7.*';
-    const LARAVEL_CURRENT_LATEST_VERSION_DEV = '7.*';
+    const LARAVEL_CURRENT_LATEST_VERSION_DEV = '8.*';
```

Comment:
why only dev?

Reply:
this is pre-relase PR, just to test it, after we test dev works, then we tag all the packages and create new PR in this installer.